heat/core/tensor.py

```diff
@@ -451,41 +451,93 @@ def __truediv__(self, other):
         """
         return arithmetics.div(self, other)
 
+    def __rtruediv__(self, other):
```

Comment:
Just a technicality, but shouldn't the functions be sorted alphabetically? (at least for the operators before it is the case)
EDIT: This goes for all the operators below